ServerCore/Pages/Components/PlayerClassSelectComponent.razor.cs

```diff
@@ -49,24 +49,24 @@ protected override async Task OnParametersSetAsync()
             {
                 SelectedPlayerClassID = NoClassSetValue;
             }
+            else
+            { 
+                SelectedPlayerClassID = CurrentTeamMember.Class?.ID ?? NoClassSetValue; 
```

Comment:
How is this possible?

Reply:
If the player registers before the event has player classes set (so it's a edge case but can still happen, mostly in testing).